frontend/src/components/documentation/Routes/Workspaces/GetSurveys.jsx

```diff
@@ -121,15 +144,35 @@ export default class WorkspacesGetSurveys extends React.Component {
                             example="false"
                             description="Whether the survey represents a staff workspace (`true`) or a student work or study space (`false`). This is useful in apps where you set the `survey_filter` parameter to `all` and wish to do filtering of student and staff workspaces in your app or API as opposed to leaving the filtering to UCL API. By default, as `survey_filter` is set to `student` unless you specify otherwise, all workspaces will have this parameter set to `false`." />
                         <Cell
-                            name="name"
+                            name="location"
+                            extra="dictionary"

```

Comment:
"dictionary" is inconsistent with the rest of the docs. "object" is preferred (and currently used), simply because we are sending JSON, not Python data structures.

Reply:
ok, I'll replace that, i forgot json calls it an object and not say dictionary, or map.